src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtils.java

```diff
@@ -512,4 +512,23 @@ public static boolean isName(String str) {
 
         return isName;
     }
+
+    /**
+     * Returns true if the value arg is either null, empty, or full of
+     * whitespace characters. More efficient that calling
+     * (string).trim().length() == 0 or string).trim().isEmpty()
+     * @param value A string to check
+     * @return true if the value is empty.
+     */
+    public static boolean isEmptyString(String value) {
```

Comment:
isn't `String` in method name redundant? We only take a string as a parameter.
I would say just name it `isEmpty`.